AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Campaign/Edit.cshtml

```diff
@@ -120,14 +120,14 @@
         <div class="form-group">
             <label asp-for="StartDate" class="col-md-2 control-label"></label>
             <div class="col-md-10">
-                <input asp-for="StartDate" class="form-control datepicker" />
+                <input type="date" asp-for="StartDate" class="form-control datepicker" />

```

Comment:
Using type="date" conflicts with the bootstrap date picker that we use in AllReady. Please remove these as it causes strange behavior with overlapping dropdowns.